### PR TITLE
[MODULAR] Makeshift Tools

### PR DESCRIPTION
**Pull request by SarmentiCampbell**
## About The Pull Request
Added Makeshift tools

## Why It's Good For The Game
Prisoners can now actually do tools to try and break out.

## Changelog
:cl:
add: Added Makeshifts tools
/:cl:

### CODE REVIEW

**Comment by tf-4** [comment]
I never understood the drill. Just let sci print regular hand drills, no need to dupe it with a slightly different sprite

File: modular_skyrat/modules/extra_tools/code/datums/components/crafting/recipes/recipes_misc.dm [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	category = CAT_MISC
+
+/datum/crafting_recipe/makeshift/hammer
+	name = "Makeshift Hammer"
+	result = /obj/item/hammer/makeshift
+	reqs = list(/obj/item/stack/cable_coil = 1,
+				/obj/item/stack/rods = 2,
+				/obj/item/kitchen/rollingpin = 2)
+	time = 80
+	category = CAT_MISC
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/extra_tools/code/datums/components/crafting/recipes/recipes_misc.dm]: What even is the purpose of a hammer...?

Comment (SarmentiCampbell) [reply modular_skyrat/modules/extra_tools/code/datums/components/crafting/recipes/recipes_misc.dm]: To be used for the other recipes.

**Comment by SarmentiCampbell** [comment]
> I never understood the drill. Just let sci print regular hand drills, no need to dupe it with a slightly different sprite
 
🤔 I honestly rather keep the green drill but we'll see regarding this

**Comment by YakumoChen** [comment]
Remove the green drill, just use the regular drill. Science doesn't need a separate item that does the exact same function and engineering doesn't need a duplicate item in their protolathe.

Not a fan of science jaws either. There's no tell that it doesn't pry doors and it goes against the behavior / expectations of all other existing jaws of life we have in the code.

Why can't scientists just ask engineering for power tools, anyway? The whole point of separating items between departments was that so science didn't just get everything.

**Comment by SarmentiCampbell** [comment]
> Remove the green drill, just use the regular drill. Science doesn't need a separate item that does the exact same function and engineering doesn't need a duplicate item in their protolathe.
> 
> Not a fan of science jaws either. There's no tell that it doesn't pry doors and it goes against the behavior / expectations of all other existing jaws of life we have in the code.
> 
> Why can't scientists just ask engineering for power tools, anyway? The whole point of separating items between departments was that so science didn't just get everything.

Take a look at the files before saying stuffs like that, please.

**Comment by YakumoChen** [comment]
In the old codebase they stacked in engineering so I don't expect it to work here either. That's still mostly overlooking the issues presented in my comment.

The green drill overlay also messes with the belt and is further reason there's no need for a duplicate.

Can you give a good reason why science needs a tool buff? Most tool use in science is minimal / doesn't have timers (namely robotics which skips all the timers) and science has no need to build many machines. I think I see medbay building more things in a given shift than science does.

**Comment by SarmentiCampbell** [comment]
> In the old codebase they stacked in engineering so I don't expect it to work here either. That's still mostly overlooking the issues presented in my comment.
> 
> The green drill overlay also messes with the belt and is further reason there's no need for a duplicate.
> 
> Can you give a good reason why science needs a tool buff? Most tool use in science is minimal / doesn't have timers (namely robotics which skips all the timers) and science has no need to build many machines. I think I see medbay building more things in a given shift than science does.

Damn, I'm not sure if you are acting dumb or not but goddamn. Check the code instead of spouting shits like that. please. 
For the green drill, Sure. Whatever. I'll make the original one be used on the science protolathe, Also. What the fuck do you mean tool buff? It is literally just a tool that can switch between screwdriver and wrench, Just because you have more spaces in your toolbelt doesn't mean it's that much of a buff 🤔

**Comment by YakumoChen** [comment]
Y-you do know that

Power tools are faster

right???

Unless you've made an undocumented nerf to power tools, they're 30% faster than regular tools, _on top of_ saving inventory space.

**Comment by SarmentiCampbell** [comment]
Oh no, Power tools are fast.

**Comment by YakumoChen** [comment]
> Can you give a good reason why science needs a tool buff? Most tool use in science is minimal / doesn't have timers (namely robotics which skips all the timers) and science has no need to build many machines. I think I see medbay building more things in a given shift than science does.

**Comment by YakumoChen** [comment]
> Why can't scientists just ask engineering for power tools, anyway? The whole point of separating items between departments was that so science didn't just get everything.

**Comment by SarmentiCampbell** [comment]
Oh no, this is so bad. I didn't expect them to be so op for science.

**Comment by YakumoChen** [comment]
Blatantly buffing a department with changes you aren't even aware of then just blowing it off like it's not important kind of tells me this PR is in pretty bad faith.

**Comment by SarmentiCampbell** [comment]
Yes, Truly. I am so saddened with what I have done.

**Comment by Azarak** [comment]
Treat people with respect or you'll suffer a gitban

Remove the power tools for science, they dont need anything like that, only engineering should have access to them. Want them? Go ask engineering

**Comment by SarmentiCampbell** [comment]
> Remove the power tools for science, they dont need anything like that, only engineering should have access to them. Want them? Go ask engineering

Only for What? For Engineering to refuse because they don't want you to be able to force doors open?

**Comment by Oscari-pog** [comment]
I do not see a single reason why science shouldn't get weaker version of powertools.
This hadn't had a single drawback in the oldbase, and it's literally just faster tools. This exists specifically so science can hand out more convenient tools to people without assistants running around with things that can force-open doors. Science is also the department that gets the most requests - ask a engineer for powertools, and he will tell you to fuck off.

**Comment by YakumoChen** [comment]
I dont see a reason to give powercreep to science in the form of better tools. Science does not typically build with tools on a given round and, as stated, Enginnering can be asked for the same thing.

 Science has more than enough power and stuff to print, give out, etc.

If engineering is telling you to fuck off, that is clearly a player issue, not a code issue. One player telling you to fuck off isnt an excuse to code a workaround to spite them.

**Review by KathrinBailey — CHANGES_REQUESTED** [review]
This is gonna be a huge pain, but...
Could you shift this to the modular_items module? It's specifically for stuff like this.
Also, it would be nice if most of these tools were weight class normal. I guess the crowbar can be bulky. If security suspect a breakout, they should be checked every nook and cranny and every storage-having item for stashed tools. 
And in regards to toolspeed, 1 is already painfully slow, so I'd put them all at 2, instead of 4, because that'll take an eternity.
I had some other stuff science could use that never got integrated into Citbase.

**Comment by Oscari-pog** [comment]
> I dont see a reason to give powercreep to science in the form of better tools. Science does not typically build with tools on a given round and, as stated, Enginnering can be asked for the same thing.
> 
> Science has more than enough power and stuff to print, give out, etc.
> 
> If engineering is telling you to fuck off, that is clearly a player issue, not a code issue. One player telling you to fuck off isnt an excuse to code a workaround to spite them.

Engineering doesnt even have a desk. They are not a department created to hand out tools when you ask for them. Science, on the other hand, has a desk that is often manned, specifically for handing out equipment per request. If you wanna change that, and give engineering a proper request desk, by all means, but as it stands right now, it is hardly powercreep - its simply faster tools. They already get the experimental welder anyhow.

**Comment by YakumoChen** [comment]
Just PDA an engineer. Desks or departments aren't always forward facing.

"Its just faster Tool speed" _is_ powercreep. That's the literal definition. 

Science actually uses the welder, it's useful for robotics. Science itself doesnt use the other tools and therefore dont get them.

**Comment by KathrinBailey** [comment]
Any further discussion of tool speed regarding power tools will be removed as it is off-topic.

**Review by Vizoplez — CHANGES_REQUESTED** [review]
Change title to include [MODULAR]

**Pull request merged**